samples/MicStream.js

```diff
@@ -0,0 +1,332 @@
+/**
```

Comment:
I see 2 different .js files for MicStreaming MicrophoneStreaming – should there be just 1? Which one is up-to-date? Will the later be removed?

Reply:
Just pushed commit d13734f - my bad, i included earlier draft - everything should be in micstream.js - running 'node micstream.js --help' lists yaaaarghs